server/views/partials/policeDetails.njk

```diff
@@ -5,10 +5,13 @@
       Name
     </dt>
     <dd class=" govuk-summary-list__value" data-qa='localPoliceForce'>
-      {{ recall.localPoliceForce }}
+      {{ getReferenceDataItemLabel('policeForces',
+        recall.localPoliceForceId
```

Comment:
Uses Id on read - as per here - not name.  Perhaps not worth further testing to prove that.

Reply:
Later removal of `localPoliceForce` name from Recall on the API will prove it is un-used on read, as intended!